README.md

```diff
@@ -4,36 +4,28 @@
 [![Contributions Welcome](https://img.shields.io/badge/contributions-welcome-brightgreen.svg?style=flat)](https://github.com/keras-team/keras-cv/issues)
 
-# Vision
-A computer vision library dedicated for auto-driving, robotics and on device applications.
 
 # Mission
 
-KerasCV is a layered repository consisting of core components and modeling components.
+KerasCV is a computer vision library of modular computer vision oriented Keras components.
+These components consist of models, layers, metrics, losses, callbacks, and utility functions.
 
-On the core components, it is made of modular building blocks (ops, functions, layers, metrics, losses, callbacks) that standardizes APIs for computer vision concepts such as data-augmentation pipeline, bounding boxes, keypoints, point clouds, feature pyramid network, etc, so applied computer vision engineers can leverage to quickly assemble production-grade, state-of-the-art
-training and inference pipelines for common tasks such as image classification, object detection and segmentation, image data augmentation, etc.
+The goal of the library is to provide standardized Keras native APIs for common computer vision tasks such as data-augmentation, classification, object detection, image generation, and more.
+Applied computer vision engineers can leverage KerasCV to quickly assemble production-grade, state-of-the-art training and inference pipelines for all of these common tasks.
 
-On the modeling components, it provides the most widely used models for each task such as ResNet family, MobileNet family, transformer-based models, anchor-based and anchor-free meta architectures, unet models, that are built on top of core components, highly composable and compatible with the Keras trainer (`model.fit`). It aims to provide pre-built models that are mixed-precision compatible, QAT compatible, and xla compilable during training, and generic model optimization tools for deployment on devices such as onboard GPUs, mobile, edge chips.
-
-KerasCV provides the following values for users:
-- modular mid-level APIs and composable meta architectures
-- mixed-precision and xla enabled components
-- highly optimized, quantization aware training (QAT) enabled models, compatible between GPUs and TPUs.
-- reproducible training results and leaderboard
-- useful tools for evaluation, visualization and explanation.
-- source for inference conversion (TFLite, edge devices, TensorRT, etc) and optimization at model level.
 
 KerasCV can be understood as a horizontal extension of the Keras API: the components are new first-party
 Keras objects (layers, metrics, etc) that are too specialized to be added to core Keras, but that receive
 the same level of polish and backwards compatibility guarantees as the rest of the Keras API and that
 are maintained by the Keras team itself.
 
+In addition to API consistency, KerasCV components are built to be mixed-precision compatible, QAT compatible,  xla compilable, and TPU compatible.
```

Comment:
Are we XLA compilable? I don't think we could claim this as we don't have XLA compilability tests.
Also I suppose that for custom components (currently custom ops) that we could not express with TF ops/API primitives we need to rely on [XLA custom calls](https://www.tensorflow.org/xla/custom_call?hl=en) to be XLA ready or not?

Reply:
Updated to say "Aims to be"